src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -1866,6 +1890,76 @@ public void recallNoticeToLoanOwnerIsSentForMovedRecallIfDueDateIsNotChanged() {
     assertThatPublishedLogRecordEventsAreValid();
   }
 
+  @Test
+  public void shouldNotCreateRequestWhenItemRequesterAndPickupServicePointAreNotProvided() {
+    Response postResponse = requestsClient.attemptCreate(new RequestBuilder()
+      .recall()
+      .withItemId(null)
+      .withRequesterId(null)
+      .withPickupServicePointId(null));
+
+    assertThat(postResponse, hasStatus(HTTP_UNPROCESSABLE_ENTITY));
+
+    final JsonObject responseJson = postResponse.getJson();
+
+    assertThat(responseJson, hasErrors(3));
+
+    assertThat(responseJson, hasErrorWith(allOf(
+        hasMessage("Cannot create a request with no item ID"),
+        hasNullParameter("itemId"))));
+
+    assertThat(responseJson, hasErrorWith(allOf(
+      hasMessage("A valid user is required. User is null."),
+      hasNullParameter("userId"))));
+
+    assertThat(responseJson, hasErrorWith(
+      hasMessage("Hold Shelf Fulfillment Requests require a Pickup Service Point")));
+  }
+
+  @Test
+  public void shouldReturnMultipleErrorsWhenRequestContainsNonExistentIds() {
```

Comment:
For similar reasons to the above tests.

```suggestion
  public void shouldNotCreateRequestWhenItemRequesterAndPickupServicePointCannotBeFound() {
```

Reply:
Renamed, thanks.